timemachine/fe/mcgregor.py

```diff
@@ -325,13 +324,17 @@ def mcs(
     base_layer = len(initial_mapping)
     priority_idxs = tuple(tuple(x) for x in priority_idxs)
     # Keep start time for debugging purposes below
-    start_time = time.time()  # noqa
-
-    mcs_result = None
+    # import time
+    # start_time = time.time()  # noqa
 
     # run in reverse by guessing max # of edges to avoid getting stuck in minima.
     max_threshold = _arcs_left(base_marcs)
+    if max_threshold < 1:
```

Comment:
nit: not for this PR but I would prefer more descriptive names for `threshold` and `max_threshold`. Are they actually numbers of edges?

Reply:
I agree. Want to leave it until another pass as I don't understand the code well enough to justify changing the naming

Reply:
max_threshold is the *maximum* possible number of common edges. Given a `NxM` mapping of edges in `mol_a` to edges in `mol_b`:
```
def _arcs_left(marcs):
    num_row_edges = np.sum(np.any(marcs, 1))
    num_col_edges = np.sum(np.any(marcs, 0))
    return min(num_row_edges, num_col_edges)
```